src/React/doc/index.rst

```diff
@@ -77,6 +77,9 @@ React components located in the directory ``assets/react/controllers`` are regis
 React controller components.
 
 You can then render any React controller component in Twig using the ``react_component``.
+The first parameter is the path to the component from the root context folder, without extension at the end.
+The second is the component props object.
```

Comment:
What about, instead of this note, we add a commented-out code example below - I'll suggest below